packages/components/src/date-range-filter-picker/README.md

```diff
@@ -6,7 +6,11 @@ Select a range of dates or single dates
 ## Usage
 
 ```jsx
-<DatePicker key={ JSON.stringify( query ) } path={ path } query={ query } />
+
+<DatePicker
```

Comment:
Just a question on why this has multiple names `DateRangeFilterPicker` & `DatePicker`. Functionally it's a `DateRangePicker`.

Reply:
I missed this - the README for this component was never published and I think the name was a placeholder.

I'd like to rename it to `DateRangePicker`, but we've already published it as `DateRangeFilterPicker`.

Reply:
Fixed in af912d14.